twiter/urls.py

```diff
@@ -0,0 +1,16 @@
+from django.urls import path
+from . import views
+from django.contrib.auth import views as auth_views
+
+app_name = 'twiter'
+
+urlpatterns = [
+     path("home/",views.home,name="home"),
+     path("list/",views.post_list,name="list"),
+     path('post/',views.tweet, name='post-create'),
```

Comment:
ここは周りに合わせて`_`にした方が良いです
```suggestion
     path('post/',views.tweet, name='post_create'),
```